api/src/main/java/org/coner/core/api/entity/CompetitionGroupApiEntity.java

```diff
@@ -66,27 +67,16 @@ public void setResultTimeType(String resultTimeType) {
     public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
-
         CompetitionGroupApiEntity that = (CompetitionGroupApiEntity) o;
-
-        if (grouping != null ? !grouping.equals(that.grouping) : that.grouping != null) return false;
-        if (handicapFactor != null ? !handicapFactor.equals(that.handicapFactor) : that.handicapFactor != null)
-            return false;
-        if (id != null ? !id.equals(that.id) : that.id != null) return false;
-        if (name != null ? !name.equals(that.name) : that.name != null) return false;
-        if (resultTimeType != null ? !resultTimeType.equals(that.resultTimeType) : that.resultTimeType != null)
-            return false;
-
-        return true;
+        return Objects.equals(id, that.id) &&
```

Comment:
Just ease of use and not having to think about equals/hashCode implementation.  Most equals/hashCode impl will be one liners.  There are cases where you might want to ignore a field and there is an overload that allows you to specify member exclusions.